marine/marine.py

```diff
@@ -86,6 +86,12 @@ def filter_and_parse(
         self._marine.marine_free(marine_result)
         return success, result
 
+    def validate_bpf(self, bpf: str) -> bool:
+        bpf = bpf.encode("utf-8")
+        if self._marine.validate_bpf(bpf) < 0:
```

Comment:
`<` or `!=`? What's more likely to remain correct?
(Not rehtorical, actually asking)

Reply:
Why doesn't it return a boolean? I see no applications for error codes here.